swift/ql/src/queries/Security/CWE-730/RegexInjection.qhelp

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Constructing a regular expression with unsanitized user input is dangerous,
+since a malicious user may be able to modify the meaning of the expression. They
+may be able to cause unexpected program behaviour, or perform a Denial of Service
```

Comment:
Not sure if we typically capitalize the 'D' and 'S' here.

```suggestion
may be able to cause unexpected program behaviour, or perform a denial-of-service
```

Reply:
In the CodeQL repo we have both "Denial of Service" and "denial-of-service" plenty of times, with little consistency.  I agree the capitalization seems a little excessive.